app/src/main/java/com/sdpteam/connectout/event/creator/LocationPicker.java

```diff
@@ -43,23 +46,27 @@ public void onMapReady(@NonNull GoogleMap googleMap) {
         map = googleMap;
 
         //Create the marker, and mark it as movable
-        MarkerOptions markerOptions = new MarkerOptions()
-                .position(new LatLng(0, 0))
-                .draggable(true)
-                .icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_BLUE));
-        movingMarker = map.addMarker(markerOptions);
+
+        LocationHelper.getInstance(getContext()).getLastLocation(getActivity(), location -> {
+            MarkerOptions markerOptions = new MarkerOptions().position(new LatLng(0, 0)).draggable(true).icon(defaultMarker(HUE_BLUE));
+            if (location != null) {
+                markerOptions.position(new LatLng(location.getLatitude(), location.getLongitude()));
+            }
```

Comment:
I found a bug, despite placing the marker at the correct position, the maps camera is not updated to focus on it.
to solve this issue I would suggest adding the following line :
                `map.moveCamera(CameraUpdateFactory.newLatLngZoom(LocationHelper.toLatLng(location), DEFAULT_MAP_ZOOM));`